### ISSUE TITLE
[Others]: Merge dev into main for a 5.2.1 version?

### ISSUE THREAD

**Issue by Cubik65536** [SlopeCraft/SlopeCraft#103]
### Description

Any plans for us to merge `dev` branch into `main` and release a `5.2.1` version (for macOS compatibility reason)?

**Comment by ToKiNoBug** [comment]
A cache-related bug hasn't been fixed in SlopeCraft, I'm working on it but I have no idea now. It takes time.

**Comment by Cubik65536** [comment]
> A cache-related bug hasn't been fixed in SlopeCraft, I'm working on it but I have no idea now. It takes time.

I don’t see where’s the issue.

**Comment by ToKiNoBug** [comment]
> > A cache-related bug hasn't been fixed in SlopeCraft, I'm working on it but I have no idea now. It takes time.
> 
> I don’t see where’s the issue.

Yes, because I discovered this bug. After fixing #99 , I found that material list can not be previewed if the image is converted with dithering in SlopeCraft. Dithering changed the original image so caches can not be loaded correctly, but in my design this should not happen. I just need some inspiration to find out the problem, and then the bug could be fixed quickly.
